### PR TITLE
fix snippets for C# 6 updates

### PR DESCRIPTION
**Pull request by BillWagner**
Various issues found while review dotnet/docs#9551

### CODE REVIEW

File: snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs [hunk 1]
```diff
@@ -159,18 +170,18 @@ public static void Main()
...
+    // </SnippetFullExample>
 
+    // <SnippetFullListExample>
     public class FullExample
     { 
-        // <SnippetFullListExample>
         class FormattedAddresses : IEnumerable<string>
         {
             private List<string> internalList = new List<string>();
             public IEnumerator<string> GetEnumerator() => internalList.GetEnumerator();
```
Comment (Thraka) [inline snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs]: Missing namespace here too?

File: snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs [hunk 2]
```diff
@@ -222,7 +233,7 @@ class RudimentaryMultiValuedDictionary<TKey, TValue> : IEnumerable<KeyValuePair<
 
             public IEnumerator<KeyValuePair<TKey, List<TValue>>> GetEnumerator() => internalDictionary.GetEnumerator();
```
Comment (Thraka) [inline snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs]: Missing namespace here too?

**Comment by Thraka** [comment]
I think you need to add those namespaces, but not sure.

File: snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs (hunk 1 above)
Comment (BillWagner) [reply snippets/csharp/programming-guide/classes-and-structs/object-collection-initializers/BasicObjectInitializers.cs]: The generic ones are loaded by default, just not the `System.Collections` namespaces.

**Pull request merged**